system/lib.py

```diff
@@ -142,6 +142,7 @@ class BaseTest(object):
         "gpgDisableSign": False,
         "ppaDistributorID": "ubuntu",
         "ppaCodename": "",
+        "databaseEtcd": "%s" % os.environ["APTLY_ETCD_DATABASE"],
```

Comment:
This will throw a `KeyError` if the env var is not set, which it isn't in the normal `system-tests`. Maybe use `os.environ.get("APTLY_ETCD_DATABASE") or ""` to make sure we have a valid value in the config?